std_msgs/Bool scan_flag
std_msgs/Bool transition_s2s
std_msgs/Bool search_flag
std_msgs/Bool recovery1_flag
std_msgs/Bool recovery2_flag
std_msgs/Bool wait
std_msgs/int32 targets_found
